test/e2e-test-application/src/app/project/project.component.html

```diff
@@ -483,6 +483,27 @@ <h3 class="fd-layout-panel__title">Navigate</h3>
             >
             </app-code-snippet>
           </li>
+          <li class="fd-list__item">
+            <a
+              href="javascript:void(0)"
+              class="fd-link"
+              (click)="
+                linkManager().openAsSplitView('/projects/pr2/webcomponent', {
+                  title: 'Webcomponent in a split view',
+                  size: '25'
+                })
+              "
+              >Open webcomponent in splitView</a
+            >
+            <app-code-snippet
+              data="linkManager()
+              .openAsSplitView('/projects/pr2/webcomponent', {
```

Comment:
I can see a space after linkmanager() which is inconvenient to copy. would be cool to see it changed at the upper openAsModal too.
```suggestion
              data="linkManager().openAsSplitView('/projects/pr2/webcomponent', {
```